motion/dp_controller/src/controller_ros.cpp

```diff
@@ -20,7 +20,9 @@ Controller::Controller(ros::NodeHandle nh) : m_nh(nh), m_frequency(10)
 {
   // Subscribers
   //m_state_sub = m_nh.subscribe("/auv/pose_gt", 1, &Controller::stateCallback, this);
-  m_state_sub         = m_nh.subscribe("/odometry/filtered", 1, &Controller::stateCallback, this);
+  // For the Qualisys ROS system at the MCLab:
+  m_state_sub	      = m_nh.subscribe("/qualisys/Body_1/odom", 1, &Controller::stateCallback, this);
+  //m_state_sub         = m_nh.subscribe("/odometry/filtered", 1, &Controller::stateCallback, this);
```

Comment:
I think this should be changed back to /odometry/filtered. Having this be selectable on launch/in the config file would be nice.

Reply:
odometry topic is now set by a ros param and has /odometry/filtered as default value